### PR TITLE
Fix vmalert relabel, update deps

### PR DESCRIPTION
**Pull request by k1rk**
* removed external label `cluster` for vmalert to fix multi cluster setup
* updated grafana dashboards
* added coredns and etcd. thanks to @kevinvirs #147 
* updated rules from upstream
* updated dependencies

### CODE REVIEW

File: charts/victoria-metrics-k8s-stack/templates/rules/serviceHealth.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ apiVersion: operator.victoriametrics.com/v1beta1
 kind: VMRule
 metadata:
   namespace: {{ .Release.Namespace }}
-  name: {{ printf "%s-%s" (include "victoria-metrics-k8s-stack.fullname" .) "servicehealth" | trunc 63 | trimSuffix "-" }}
+  name: {{ printf "%s-%s" (include "victoria-metrics-k8s-stack.fullname" .) "serviceHealth" | trunc 63 | trimSuffix "-" }}
```
Comment (f41gh7) [inline charts/victoria-metrics-k8s-stack/templates/rules/serviceHealth.yaml:11]: serviceHealth must be in lower case?

Comment (k1rk) [reply charts/victoria-metrics-k8s-stack/templates/rules/serviceHealth.yaml:11]: fixed

**Review by f41gh7 — APPROVED** [review]
LGTM

**Comment by f41gh7** [comment]
Thanks!

**Pull request merged**